shared/messaging/events.go

```diff
@@ -9,3 +11,25 @@ const (
 	// Utility
 	TxGossipMessageContentType = "utility.TxGossipMessage"
 )
+
+// Helper logger for state sync tranition events
+func EventToMap(stateSyncMsg *StateMachineTransitionEvent) map[string]any {
+
+	fields := map[string]any{
+		"previous_state": stateSyncMsg.PreviousState,
+		"new_state":      stateSyncMsg.NewState,
+	}
+
+	switch types.StateMachineEvent(stateSyncMsg.Event) {
+	case types.StateMachineEvent_P2P_IsBootstrapped:
+		fields["proto_type"] = "StateMachineEvent_P2P_IsBootstrapped"
```

Comment:
1. These are not `proto_types`, they are the `state_machine_event`
2. Rather than having a switch statement, you can just cast the event to the string in a one line (within the map) because: `type StateMachineEvent string` in `fsm_events.go`

```go
fields := map[string]any{
		"previous_state": stateSyncMsg.PreviousState,
		"new_state":        stateSyncMsg.NewState,
		"state_machine_event":      string(stateSyncMsg.Event),
	}
```